manage.sh

```diff
@@ -17,23 +17,41 @@ if [ "$command" == "install" ]; then
     source .venv/bin/activate
     pip install --upgrade pip
     pip install -r requirements.txt
-elif [ "$command" == "start" ]; then
+    set_env=true
+    if [ -f env_vars ]; then
+        read -p "Firebase URL and bot token already set - reset them? (y/n): " reset_env
+        if [ $reset_env != "y" ]; then set_env=false; fi
+    fi
+    if $set_env; then
+        echo "==========================================================================="
+        read -p "Please enter the IP adress of the server (defaults to localhost): " server_ip
+        read -p "Please enter the token of your bot (received from the telegram bot father): " bot_token
+        echo "==========================================================================="
+        server_url=$([ $server_ip == "" ] && echo localhost || echo "http://$server_ip:8080")
+        echo "export GP_SERVER_URL=$server_url" > env_vars
+        echo "export GP_TELEGRAM_BOT_TOKEN=$bot_token" >> env_vars
+    fi
+    exit
+fi
+
+# try to load venv
+if [ -d .venv ]; then
+    source .venv/bin/activate
+else
+    echo "please first install the bot"
+    exit
+fi
+[ -f env_vars ] && source env_vars
```

Comment:
Dont these two lines conflict each other?

Reply:
no, why would they? The `source` command just executes the given shell script, but not in a subshell as usual but directly in the calling shell. So you can source as many scripts as you want